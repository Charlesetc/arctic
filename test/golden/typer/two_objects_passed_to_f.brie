define main [ [
        define f : x [
                x.this 
                x
        ]
        f < this = 2 , that = 3>
        f < wow = 3 , this = 9> 
]            ]
