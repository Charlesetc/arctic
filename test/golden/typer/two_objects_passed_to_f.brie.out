root[(ident(define):: ident(main):: block[][(block[][(ident(define):: ident(f):: block[ident(x)::][(ident(x)::.this::)::, (ident(x)::)::]::)::, (ident(f):: <this = (ident(2)::):: , that = (ident(3)::)::>::)::, (ident(f):: <wow = (ident(3)::):: , this = (ident(9)::)::>::)::]::Function)::Function]::Function)::]::